pkg/dataMover/data_mover.go

```diff
@@ -31,17 +32,19 @@ type DataMover struct {
 	s3PETM  *s3repository.ProtectedEntityTypeManager
 }
 
-func NewDataMoverFromCluster(logger logrus.FieldLogger) (*DataMover, error) {
-	params := make(map[string]interface{})
-	err := utils.RetrieveVcConfigSecret(params, logger)
+func NewDataMoverFromCluster(params map[string]interface{}, logger logrus.FieldLogger) (*DataMover, error) {
```

Comment:
Maybe a more specific variable name than params?

Reply:
Updated the variable name from params to configParams in server.go - newServer() function. Will retain the name param in NewDataMoverFromCluster() as this was the existing variable name, used at multiple places.